library/model/src/main/java/org/cru/godtools/model/Language.java

```diff
@@ -34,6 +39,15 @@ public void setCode(@Nullable final Locale code) {
         mCode = code;
     }
 
+    @NonNull
+    public String getLanguageName() {
+        return mLanguageName;
+    }
+
+    public void setLanguageName(String name) {
```

Comment:
annotate `String name` with `@Nullable` so that kotlin knows whether or not a null value is acceptable